docs/admin/users/index.md

```diff
@@ -195,6 +197,8 @@ The following filters are supported:
 - `role` - Represents the role of the user. You can refer to the
   [TemplateRole documentation](https://pkg.go.dev/github.com/coder/coder/v2/codersdk#TemplateRole)
   for a list of supported user roles.
-- `last_seen_before` and `last_seen_after` - The last time a used has used the
+- `last_seen_before` and `last_seen_after` - The last time a user has used the
   platform (e.g. logging in, any API requests, connecting to workspaces). Uses
   the RFC3339Nano format.
+- `created_at_before` and `created_at_after` - The time a user was created. Uses
```

Comment:
I don't think `created_at_{after|before}` reads very well. I didn't really think about the phrasing when I wrote it earlier. Let's just go with `created_before` and `created_after`.

Reply:
@ethanndickson 

updated the PR!